### PR TITLE
Issue 2013

### PR DESCRIPTION
**Pull request by pointnet**
#2013

### CODE REVIEW

**Comment by pointnet** [comment]
Appveyor failed because it use react 15.2.0, do I need to change the package.json in the PR ?

**Comment by taion** [comment]
Looks fine to me.

File: src/DropdownToggle.js [hunk 1]
```diff
@@ -31,11 +40,13 @@ export default class DropdownToggle extends React.Component {
 DropdownToggle.defaultProps = {
   open: false,
   useAnchor: false,
-  bsRole: 'toggle'
+  bsRole: 'toggle',
+  bsClass: 'dropdown'
```
Comment (jquense) [inline src/DropdownToggle.js]: for consistency this should be defined via the bsClass helper in bootstrapUtils, other than that looks good :thumbsup:

Comment (pointnet) [inline src/DropdownToggle.js]: fixed

Comment (taion) [inline src/DropdownToggle.js]: I've actually been going through and swapping everything out for `bsClass` on my "update for React v15.2.0" branch.

**Comment by jquense** [comment]
LGTM, can't merge from my phone tho

**Pull request merged**